packages/fether-react/src/stores/createAccountStore.js

```diff
@@ -137,13 +157,19 @@ export class CreateAccountStore {
   /**
    * Set phrase (detect type) and corresponding address
    */
-
   setPhrase = phrase => {
     return this.setBip39Phrase(phrase).catch(() =>
```

Comment:
.toLowerCase() on those two lines I think to allow for case-insensitive Parity wordlist / bip 39 input